### DISCUSSION TITLE
viewer breaks after rotating on ios 14

### DISCUSSION THREAD

**Discussion by corecode1** [google/model-viewer#2212] [Q&A]
### Description

Wait till the page and model loads. Then rotate device. Model viewer reports webgl context lost error, and nothing is displayed. Model-viewer must be fullscreen. This happens only with thin models, adding more thickness to exported mesh or additional thicker objects to model fixes the problem.

Happens on IOS 14 Public beta 2 and 3. Issue occured on Iphone 11 in both safari and chrome. 
Not reproduced on ios 13, android and pc.

#### Live Demo
edit:
https://glitch.com/edit/#!/join/f6d6cdbb-252e-467d-b280-bc5ccd7c435d
test from mobile device: 
https://model-viewer-issue.glitch.me

### Version
- model-viewer: v1.1.0

### Browser Affected
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari 

### OS
- [ ] Android
- [x] iOS 14 Public Beta 2, 3
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Wow, that's a strange one. So context loss here is dependent on both model geometry and the phone changing orientation? I have a hard time believing this is a model-viewer bug. Have you filed it on Safari?

**Comment by alexkowel** [comment]
Looks similar to [this](https://forum.babylonjs.com/t/context-lost-webgl-in-ios-14-0/14549) issue in Babylon.js .

**Comment by corecode1** [comment]
Sorry for not responding so long. I didn't file it in Safari because in my app I have threejs fullscreen canvas as well, and it works well after rotating, so I thought there must me something in model viewer. By the way, it still happens in non-dev iOS 14.0.1
